Sources/GRPCInteropTests/InteroperabilityTestCases.swift

```diff
@@ -707,13 +707,13 @@ struct CustomMetadata: InteroperabilityTest {
   let trailingMetadataValue: [UInt8] = [0xAB, 0xAB, 0xAB]
 
   func checkInitialMetadata(_ metadata: Metadata) throws {
```

Comment:
Should we rename this to `checkStringValuedMetadata` (or similar) and the next method `checkBinaryValuedMetadata` instead?

Reply:
I don't think so: we are checking the initial and trailing metadata are the ones expected by the test.

Reply:
As in, the test (description) expects that a particular set of initial metadata and a particular set of trailing metadata.